test/integration/068_partial_parsing_tests/test-files/people_metrics.yml

```
@@ -0,0 +1,30 @@
+version: 2
+
+metrics:
+
+  - model: people
```

Here and below, this should be:
```suggestion
  - model: ref('people')
```
I had to use it to understand it :)

---

Fixed.